scripts/run_benchmarks.sh

```diff
@@ -183,7 +183,7 @@ function run_benchmark() {
   --chain="frequency-bench" \
   --heap-pages=4096 \
   --wasm-execution=compiled \
-  --additional-trie-layers=20 \
+  --additional-trie-layers=5 \
```

Comment:
Question: why `5`?

Answer: 

So my understanding is that during PoV benchmarks we have two steps of computation
1- PoV from Main root node to the root of desired storage map.
2- PoV for storageMap

`additional-trie-layers` is for the first step and it's the LOG16(total number of storage items). By default this number should be between 2-3 and we don't need to set that. But since we are using child trees and it seems like child-trees are not getting benchmarked the same way that StorageMaps might have, **to be cautious** we are setting this to 5 which means it should be fine for at least 1_000_000 child trees. in case the number of child trees passed 1 Million this might need to be bumped to 6.

 for second step if `worst-case-map-values` is not set it will use 1_000_000 as the number of items in map.